skyreader/plot/plot.py

```diff
@@ -133,50 +139,79 @@ def create_plot(self, result: SkyScanResult, dozoom: bool = False) -> None:
             del this_map
             del grid_pix
 
-        LOGGER.info(f"min  RA: {min_ra *180./np.pi} deg, {min_ra*12./np.pi} hours.")
+        LOGGER.info(
+            f"min  RA: {min_ra * 180./np.pi} deg, {min_ra*12./np.pi} hours."
+        )
         LOGGER.info(f"min Dec: {min_dec * 180./np.pi} deg")
 
         # renormalize
         if dozoom:
             grid_map = grid_map - min_llh
-            # max_value = max_value - min_value
             min_llh = 0.
             max_llh = 50
-
+        if not llh_map:
+            prob_map = (copy.copy(grid_map) - min_llh)/2.
+            prob_map = -prob_map*np.log10(np.exp(1))
+            min_prob = np.nanmin(prob_map)
+            max_prob = np.nanmax(prob_map)
+            prob_map = np.ma.masked_invalid(prob_map)
         grid_map = np.ma.masked_invalid(grid_map)
 
         LOGGER.info(f"Preparing plot: {plot_filename}...")
 
         # the color map to use
-        cmap = self.PLOT_COLORMAP
-        cmap.set_under(alpha=0.) # make underflows transparent
-        cmap.set_bad(alpha=1., color=(1.,0.,0.)) # make NaNs bright red
+        if llh_map:
```

Comment:
It could be nice to merge these conditionals on the `llh_map` bool above and below.